### ISSUE TITLE
environment setup details

### ISSUE THREAD

**Issue by sahithkumar1994** [halloTheCoder/car-damage-evaluation-using-deep-learning#3]
can you add the version details of tensorflow,  keras, and other packages

**Comment by Adesoji1** [comment]
@sahithkumar1994  This is the requirement.txt for the webapp.


absl-py==2.1.0
astunparse==1.6.3
cachetools==5.3.2
certifi==2023.11.17
charset-normalizer==3.3.2
clang==5.0
click==8.1.7
contourpy==1.1.1
cycler==0.12.1
Flask==2.0.3
flatbuffers==1.12
fonttools==4.47.2
gast==0.4.0
google-auth==2.27.0
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
grpcio==1.60.0
h5py==3.1.0
idna==3.6
importlib-metadata==7.0.1
importlib-resources==6.1.1
itsdangerous==2.1.2
Jinja2==3.1.1
keras==2.9.0
Keras-Preprocessing==1.1.2
kiwisolver==1.4.5
libclang==16.0.6
Markdown==3.5.2
MarkupSafe==2.1.4
matplotlib==3.7.4
numpy==1.24.4
oauthlib==3.2.2
opt-einsum==3.3.0
packaging==23.2
pillow==10.2.0
protobuf==3.19.0
pyasn1==0.5.1
pyasn1-modules==0.3.0
pyparsing==3.1.1
python-dateutil==2.8.2
requests==2.31.0
requests-oauthlib==1.3.1
rsa==4.9
six==1.15.0
tensorboard==2.9.1
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow-estimator==2.9.0
tensorflow-gpu==2.9.0
tensorflow-io-gcs-filesystem==0.34.0
termcolor==1.1.0
typing-extensions==3.7.4.3
urllib3==2.1.0
Werkzeug==2.3.0
wrapt==1.12.1
zipp==3.17.0
